skbeam/io/fit2d.py

```diff
@@ -17,7 +17,7 @@ def fit2d_save(mask, filename, dir_path=None):
     dir_path: str, optional
         Path to the destination file
     """
-    saver = Fit2dMaskImage(data=~mask)
+    saver = Fit2dMaskImage(data=~np.flipud(mask))
```

Comment:
Please put a note in function doc to mention this. It looks like a fix is needed in fabio?

Reply:
yes explanation would help? why is this needed?